pkg/pbflow/proto.go

```diff
@@ -98,9 +104,17 @@ func FlowToPB(fr *model.Record, s *ovnobserv.SampleDecoder) *Record {
 	if fr.Metrics.EthProtocol == model.IPv6Type {
 		pbflowRecord.Network.SrcAddr = &IP{IpFamily: &IP_Ipv6{Ipv6: fr.ID.SrcIp[:]}}
 		pbflowRecord.Network.DstAddr = &IP{IpFamily: &IP_Ipv6{Ipv6: fr.ID.DstIp[:]}}
+		if fr.Metrics.AdditionalMetrics != nil {
```

Comment:
in dual stack is it possible to have traffic over ipv4 NATed to ipv6, or the other way around? In other words, are we sure that the EthProtocol that we got in the first place is still the same in xlat ?
If not, then we don't know here if the xlat addresses are v4 or v6 ?

Reply:
in theory its possible but I don't think k8s service will change from v4 to v6 but we can fix that when this happen ?